pkg/grafana/client/client.go

```diff
@@ -0,0 +1,51 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+
+	"github.com/go-logr/logr"
+	grafana "github.com/grafana/grafana-openapi-client-go/client"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	GrafanaUrl           = "http://grafana.monitoring.svc.cluster.local:3000"
```

Comment:
```suggestion
	grafanaURL           = "http://grafana.monitoring.svc.cluster.local:3000"
```

Reply:
Maybe the monitoring part can be replace with a %s and be templated from the value of the namespace const?